support/cas-server-support-inwebo-mfa/src/main/java/org/apereo/cas/support/inwebo/web/flow/actions/CheckAuthenticationAction.java

```diff
@@ -0,0 +1,72 @@
+package org.apereo.cas.support.inwebo.web.flow.actions;
+
+import org.apereo.cas.support.inwebo.authentication.InweboCredential;
+import org.apereo.cas.support.inwebo.service.InweboService;
+import org.apereo.cas.support.inwebo.service.response.Result;
+import org.apereo.cas.web.flow.resolver.CasWebflowEventResolver;
+import org.apereo.cas.web.support.WebUtils;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.webflow.action.AbstractAction;
+import org.springframework.webflow.execution.Event;
+import org.springframework.webflow.execution.RequestContext;
+
+/**
+ * A web action to check the push notification or the OTP generated on the browser.
+ *
+ * @author Jerome LELEU
+ * @since 6.4.0
+ */
+@RequiredArgsConstructor
+@Slf4j
+public class CheckAuthenticationAction extends AbstractAction implements WebflowConstants {
+
+    private final MessageSource messageSource;
+
+    private final InweboService service;
+
+    private final CasWebflowEventResolver casWebflowEventResolver;
+
+    @Override
+    public Event doExecute(final RequestContext requestContext) {
+        val authentication = WebUtils.getInProgressAuthentication();
+        val login = authentication.getPrincipal().getId();
+        LOGGER.debug("Login: {}", login);
```

Comment:
This should be a trace statement. Also please use `[{}]` for placeholders to be consistent.

Reply:
OK. Will change also.